python/ray/air/tests/test_tensor_extension.py

```diff
@@ -13,9 +13,24 @@
     ArrowVariableShapedTensorType,
 )
 from ray.air.util.tensor_extensions.pandas import TensorArray, TensorDtype
+from ray.air.util.tensor_extensions.utils import create_possibly_ragged_ndarray
 from ray._private.utils import _get_pyarrow_version
 
 
+@pytest.mark.parametrize(
+    "values",
+    [
+        [np.zeros((3, 1)), np.zeros((3, 2))],
+        [np.zeros((3,))],
+    ],
+)
+def test_create_possibly_ragged_ndarray(values):
+    ragged_array = create_possibly_ragged_ndarray(values)
+
+    assert [array.shape for array in ragged_array] == [array.shape for array in values]
```

Comment:
nit: here, is it possible to use `np.testing.assert_array_equal` to compare the shape, dtype, and values of the arrays simultaneously?
https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_equal.html